doc/index.md

```diff
@@ -17,6 +17,9 @@ For Developers
 
 -   [Build Objective-C (iOS) WebSocket Clients](dev-objc/o_dev_objc.md)
 -   [Build Android and Java WebSocket Clients](dev-android/o_dev_android.md)
```

Comment:
@vjwang and @michaelcretzman Your comments have been addressed. Good suggestions. Thanks for your help.